Mathlib/GroupTheory/GroupAction/Quotient.lean

```diff
@@ -346,6 +346,91 @@ instance isPretransitive_quotient (G) [Group G] (H : Subgroup G) : IsPretransiti
 #align mul_action.is_pretransitive_quotient MulAction.isPretransitive_quotient
 #align add_action.is_pretransitive_quotient AddAction.isPretransitive_quotient
 
+variable {α}
+
+@[to_additive]
+lemma finite_quotient_of_pretransitive_of_finite_quotient [IsPretransitive α β] {H : Subgroup α}
```

Comment:
Why not `[Finite (α ⧸ H)]`? Also, can this be an instance?

Reply:
Changed to an instance and to use instance argument.